docker-compose.yml

```diff
@@ -95,23 +93,17 @@ services:
     image: dlrriesgos/frontend
     depends_on:
       - backend
-    ports:
-      - 8000:80
     environment:
-      subPath: ${frontendSubPath}
+      subPath: "/frontend/"
       backendUrl: ${backendUrl}
-      backendPort: ${backendPort}
 
   compare-frontend:
     image: dlrriesgos/compare-frontend
     depends_on:
       - backend
-    ports:
-      - 8001:80
     environment:
-      subPath: ${compareFrontendSubPath}
```

Comment:
Same here. We may not need to have this still in the .env file, nor in the readme.md.

Reply:
Removed from .env!